### DISCUSSION TITLE
code不生效

### DISCUSSION THREAD

**Discussion by lastbaba** [ChatGPTNextWeb/NextChat#1884] [Q&A]
有个问题，不知道大家有没有遇到过，我在自己的服务器上部署以后，在环境变量里设置好code以后，在网页的是设置页面里面填了code，但是回复的是个错误
{
  "error": {
    "message": "",
    "type": "invalid_request_error",
    "param": null,
    "code": "invalid_api_key"
  }
}
如果我填可OpenAI API Key，就不会有问题，这会是什么问题呢，总能看到有提示说访问密码是空的，我在设置里填写的密码是和环境变量里面的完全一样的值，但是网页端貌似没有识别到这个内容。

有大神知道具体怎么处理吗？

**Comment by adevjoe** [comment]
新建一个对话好像可以

**Reply by lastbaba** [reply]
我试过新建会话也不行，输入密码完全没有效果

**Reply by steveny1989** [reply]
遇到同样问题。求解决。
